CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.0.1-dev
+
+* Handle `expires_in` when encoded as string.
```

Comment:
Did we miss a changelog in an earlier PR? Can you link it in the commit message?

Reply:
Yup - here: https://github.com/dart-lang/oauth2/commit/8b4f47832c56a02e0143d6741d57fd2c6b0063b0. I'll add a reference to the commit when merging this.